confgenerator/testdata/invalid/linux/logging-processor_exclude_logs_invalid_path_cr/golden_error

```diff
@@ -1,4 +1,4 @@
-[5:18] "match_any[0]": 1:25: error: target may not contain line breaks, spaces, commas, or double-quotes: a\rb
+[5:18] "match_any[0]": 1:25: error: target may not contain line breaks, spaces, commas, or double-quotes: a
b
```

Comment:
I can't immediately tell which change caused this, but is there any chance we can get the escaping back in these error messages? Not a blocker but the `\r` is much easier to see when it's escaped.

Reply:
I agree, it was the change to make `fluentBitPath` return an unescaped path.